### ISSUE TITLE
Validate ground truth labels

### ISSUE THREAD

**Issue by kutayeroglu** [kutayeroglu/ijepa#32]

**Comment by kutayeroglu** [comment]
The error's cause is the flat structure of the validation set.
It was most likely downloaded from Kaggle, see the following link for people having a similar problem: https://discuss.pytorch.org/t/issues-with-dataloader-for-imagenet-should-i-use-datasets-imagefolder-or-datasets-imagenet/115742

I need to re-organize the validation data.
Instead of re-organizing the shared dataset, I will download the validation data and reorganize it in my personal storage area.
